src/encoded/schemas/library.json

```diff
@@ -155,6 +165,11 @@
             "type": "string",
             "format": "date"
         },
+        "target_dna_sequence": {
+            "title": "Target DNA Sequence",
+            "description": "Target DNA sequence used in insert selection",
```

Comment:
Was this property also intended for the Cas9 capture experiments? It looks like we added a "dna_target" property on the UMich metadata sheet (https://docs.google.com/spreadsheets/d/1sFOEEc_JtE-wNO7RnN4EcaEVeBuDU8uMNHnAJzoLc2s/edit?gid=493168297#gid=493168297), but I wasn't sure if this property was intended to be used in the same way.

Reply:
I see, I wasn't sure exactly what the intended values were for this. Updating!